upstream_dependencies.rosinstall

```diff
@@ -1,4 +1,5 @@
 # These are upstream source dependencies used by all builds.
 
 # When this file starts getting used, add it to the options(UPSTREAM_WORKSPACE: "upstream_dependencies.rosinstall") of the following CI stages:
-arm64_build , amd64_build, clang-tidy, catkin_lint
+# arm64_build , amd64_build, clang-tidy, catkin_lint
+#
```

Comment:
remove this #

Reply:
didnt you want me to add this? "also in upstream_dependencies.rosinstall can u add a "#" on the last lane"

Reply:
just the one on line 4, not on line 5